test-projects/access-control/CHANGELOG.md

```diff
@@ -24,7 +24,7 @@
       ```diff
       +const express = require('express');
        const port = 3000;
-       keystone.prepare({ port })
```

Comment:
Is the CHANGELOG changes necessary?
I haven't done it for every change log [yet]. There were quite a few.

Reply:
No, the `CHANGELOG`s are historical records and shouldn't be edited.

Reply:
Removed